acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/web3j/privacy/OnChainPrivacyAcceptanceTest.java

```diff
@@ -100,8 +101,9 @@ public void canAddParticipantToGroup() {
 
     charlie.verify(privateTransactionVerifier.noPrivateTransactionReceipt(commitmentHash));
 
-    lockPrivacyGroup(privacyGroupId, alice);
-    addMembersToPrivacyGroup(privacyGroupId, alice, charlie);
+    lockPrivacyGroup(privacyGroupId, alice, Credentials.create(alice.getTransactionSigningKey()));
```

Comment:
NIT: we could assign `Credentials.create(alice.getTransactionSigningKey())` to a variable `aliceCredentials` to make the code easier to read.

Reply:
Done